newdle_frontend.js

```diff
@@ -40,6 +40,12 @@ function onSubmit(){
 
 // Functions otherwise
 
+GUESS_BOX.addEventListener("keypress", function(event){
+    if (event.key === "Enter"){
+        document.getElementById("submit_button").click();
```

Comment:
The other way of doing this would be to just call the function that the submit button calls.
But I think this is the better approach because it means if you change the function that the submit button calls then this will also update